app/components/chart/header.js

```diff
@@ -15,21 +15,24 @@
  * not, you can obtain one from Tidepool Project at tidepool.org.
  * == BSD2 LICENSE ==
  */
-var bows = require('bows');
-var React = require('react');
-var cx = require('classnames');
+import _ from 'lodash';
+import bows from 'bows';
+import React from 'react';
+import cx from 'classnames';
+import Loading from 'react-loading';
 
-var PrintHeader = require('../printheader');
+import PrintHeader from '../printheader';
 
-var printPng = require('./img/print-icon-2x.png');
+import printPng from './img/print-icon-2x.png';
 
-var tideline = {
-  log: bows('Header')
+const tideline = {
+  log: bows('Header'),
 };
 
-var TidelineHeader = React.createClass({
+const TidelineHeader = React.createClass({
```

Comment:
Will do.  Wasn't sure if I was to take on converting it over, though I agree, if I'm going to ES6-ify some of the file, might as well go all the way :)